dymos/examples/finite_burn_orbit_raise/finite_burn_orbit_raise_problem.py

```diff
@@ -221,7 +221,7 @@ def two_burn_orbit_raise_problem(transcription='gauss-lobatto', optimizer='SLSQP
         p.driver.opt_settings['compl_inf_tol'] = 1e-3
         p.driver.opt_settings['acceptable_iter'] = 0
         p.driver.opt_settings['tol'] = 1e-3
-        p.driver.opt_settings['print_level'] = 5 if show_output else 0
+        p.driver.opt_settings['print_level'] = 0 if show_output else 0
```

Comment:
This is a little picky but if we're going to offer the `show_output` option it should do something.  Either leave this as it was and make sure we call it with `show_output = False`, or just remove it as an argument altogether.

Reply:
Oops that was just an oversight.  I was doing a search and replace and didn't catch that that was part of a conditional.  Fixed.

Reply:
I figured that was the case.